### ISSUE TITLE
unlootable items in Solstheim

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#168]
Nexus report

> Ash Pile ACTI 0403280a
> Pile of Gold ACTI 04018486
> Pile of Gold ACTI 04018488
> Also Ice Wraith Pods and a couple of other "Essence Blob in Lieu of a Corpse" containers
> 

Piles of Gold are found in Gyldenhul Barrow

**Comment by SteveTownsend** [comment]
This looks like the ACTIs just need to be added as _septims_

**Comment by SteveTownsend** [comment]
This is fixed for v2.5.0.0. Test evidnce:
```
2020-08-13 12:36:40.235    trace  32504 Process REFR 0xff001ebe with base object Pile of Septims/0x04018486
2020-08-13 12:36:40.236    trace  32504 SmartHarvest Pile of Septims/0x04018486 for REFR 0xff001ebe, collectible=false
2020-08-13 12:36:40.236    trace  32504 Process REFR 0xff001ebf with base object Pile of Septims/0x04018488
2020-08-13 12:36:40.236    trace  32504 SmartHarvest Pile of Septims/0x04018488 for REFR 0xff001ebf, collectible=false
```
